tests/Sniffs/Complexity/CognitiveSniffTest.php

```diff
@@ -99,19 +99,22 @@ public function testNoErrors(string $filepath, int $line, string $functionName,
 			'maxComplexity' => $expectedComplexity,
 		]);
 		self::assertNoSniffErrorInFile($report);
+		self::assertNoSniffWarningsInFile($report);
```

Comment:
Please use the new options here. And create to new tests (errors and no-error) for the deprecated option.